src/build.gradle

```diff
@@ -3,9 +3,9 @@ buildscript {
         jcenter()
     }
```

Comment:
Ran into some issues with build. Not sure if related to this task, but mentioning it anyway. For me this is reproducible.

1. clean proxy-ui-api module

```
~/p/n/x/g/X/s/proxy-ui-api (XRDDEV-1085 ⚡↩☡=) ../gradlew clean

BUILD SUCCESSFUL in 2s
1 actionable task: 1 executed

```

2. build root project -> one proxy-ui-api test fails

```
~/p/n/x/g/X/src (XRDDEV-1085 ⚡↩☡=) ./gradlew build

> Task :proxy-ui-api:test

org.niis.xroad.restapi.openapi.ServiceDescriptionsApiControllerIntegrationTest > refreshServiceDescription FAILED
    java.lang.AssertionError at ServiceDescriptionsApiControllerIntegrationTest.java:250
```


3. build root project again -> proxy-ui-api tests now all pass. This time messagelog test fails.

```
> Task :addons:messagelog:test

ee.ria.xroad.proxy.messagelog.MessageLogTest > logTimestampArchiveAndClean FAILED
    java.lang.RuntimeException at MessageLogTest.java:514

```

Reply:
I was not able to reproduce the problem.